evap/staff/tests/test_importers.py

```diff
@@ -13,14 +13,58 @@ class TestUserImporter(TestCase):
     filename_invalid = os.path.join(settings.BASE_DIR, "staff/fixtures/invalid_user_import.xls")
     filename_random = os.path.join(settings.BASE_DIR, "staff/fixtures/random.random")
 
+    # valid user import tested in tests/test_views.py, TestUserImportView
+
+    @classmethod
+    def setUpTestData(cls):
+        with open(cls.filename_valid, "rb") as excel_file:
+            cls.valid_excel_content = excel_file.read()
+        with open(cls.filename_invalid, "rb") as excel_file:
+            cls.invalid_excel_content = excel_file.read()
+        with open(cls.filename_random, "rb") as excel_file:
+            cls.random_excel_content = excel_file.read()
+
+    def test_test_run_consistency(self):
+        original_users = list(UserProfile.objects.all())
+
+        __, __, __, __ = UserImporter.process(self.valid_excel_content, test_run=True)
+
+        self.assertEqual(original_users, list(UserProfile.objects.all()))
+
+    def test_test_and_notest_equality(self):
+        # success messages are supposed to be different
+        list_test, __, warnings_test, errors_test = UserImporter.process(self.valid_excel_content, test_run=True)
+        list_notest, __, warnings_notest, errors_notest = UserImporter.process(self.valid_excel_content, test_run=False)
+
+        notest_string_list = ["{} {}".format(user.full_name, user.email) for user in list_notest]
+        for user in list_test:
+            self.assertIn("{} {}".format(user.full_name, user.email), notest_string_list)
```

Comment:
scratch that, what about making such a string_list for both `list_test`and `list_notest` and comparing these lists for equality?